build.rs

```diff
@@ -70,7 +70,8 @@ fn bindgen() {
         .expect("Unable to generate bindings");
 
     // Write the bindings to the $OUT_DIR/$BINDINGS file.
-    let out_path = std::path::PathBuf::from(std::env::var("OUT_DIR").unwrap()).join(BINDINGS);
+    let out_path =
+        std::path::PathBuf::from(std::env::var("OUT_DIR").unwrap_or_default()).join(BINDINGS);
```

Comment:
If `OUT_DIR` isn't set, this would write to `./bindings.rs`?

Reply:
Yeah. A panic is probably better.